### DISCUSSION TITLE
M1 mac error

### DISCUSSION THREAD

**Discussion by bezdonas** [Try/OpenGothic#377] [Q&A] [answered]
I've cloned and built OpenGothic as is instruction from README.md.

Installed Gothic classic GOG version using PortingKit(wine wrapper)

Upon trying 
`./Gothic2Notr -g /Users/ramilaliev/Applications/Gothic 2 Gold.app/Contents/SharedSupport/prefix/drive_c/GOG Games/Gothic 2 Gold`

I get this error:
```
./Gothic2Notr -g /Users/ramilaliev/Applications/Gothic 2 Gold.app/Contents/SharedSupport/prefix/drive_c/GOG Games/Gothic 2 Gold
dyld[83983]: Library not loaded: @rpath/libTempest.dylib
  Referenced from: /Users/ramilaliev/Desktop/Gothic 2 classic/opengothic/OpenGothic/build/opengothic/Gothic2Notr
  Reason: tried: '/usr/local/lib/libTempest.dylib' (no such file), '/usr/lib/libTempest.dylib' (no such file)
[1]    83983 abort      ./Gothic2Notr -g /Users/ramilaliev/Applications/Gothic 2  Games/Gothic 2 Gold
```

**Comment by Try** [comment, marked as answer]
Hi, @bezdonas !

You need to use wrapped script `GothicNotr.sh` to run the game. Or, alternatively add `libTempest` to you dylib path

**Reply by bezdonas** [reply]
That worked, thanks!
I'm amazed by the work you did here to be honest.
